lib/oli_web/components/layouts/instructor_dashboard.html.heex

```diff
@@ -1,4 +1,4 @@
-<div class="flex-1 flex flex-col h-screen">
+<div class="flex-1 flex flex-col h-auto">
```

Comment:
`h-auto` seems to be the reason why we see the cookie preferences for a second. I think it may have to do with liveview lifecycle, when the "dead" view gets rendered the height of the screen is shorter, since the other content has not yet been rendered. 
Why do we need this change? reverting it seems to fix the "cookie preferences" issue

Reply:
That's the point. We could go with `h-auto` that is going to bring the Cookie Preferences issue, or what you mentioned:

> With this approach, we get an animation when scrolling all the way up to the top, where a white background is seen by a fraction of a sec. (maybe [overscroll-behaviour ](https://developer.mozilla.org/en-US/docs/Web/CSS/overscroll-behavior)can fix it, not sure TBH).

Either way, something looks weird.

Reply:
I [tried](https://github.com/Simon-Initiative/oli-torus/pull/4808/commits/aec24579f2b22d229ac03b43e1755aa80cc8d5a9) using `min-h-screen`, and it did the trick.